bridge/test/contract-mocks/utils/MockMerkleProofLibrary.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT-open-group
```

Comment:
Great question! The issue is that the underlying library has its functions as internal, thus calling the functions directly is not possible without exposing the functions, and in general altering a class for the sake of testing should be avoided. It could be argued that there is value in having them as public functions however even after exposing them and directly deploying the library the failure test scenarios were not failing with the expected reasons, they were failing with messages around the library being called directly e.g.:

```
AssertionError: Expected transaction to be reverted with MerkleProofLibrary: proofHeight should be in the range [0, 256], but other exception was thrown: Error: Transaction reverted: library was called directly
```

I'm open to suggestions for other ways of testing the functions in isolation if you have any!